back/app/Http/Controllers/MessageController.php

```diff
@@ -0,0 +1,65 @@
+<?php
+
+namespace App\Http\Controllers;
+
+use Illuminate\Http\Request;
+use App\Http\Requests\MessageRequest;
+use App\Http\Requests\UpdateMessageActivationRequest;
+use App\Models\Message;
+use App\Http\Resources\MessageResource;
+use Illuminate\Http\JsonResponse;
+
+class MessageController extends Controller
+{
+    public function index()
+    {
+        $this->authorize('index', Message::class);
+        return response()->json(MessageResource::collection(Message::all()));
+    }
+
+    public function store(MessageRequest $request)
+    {
+        $this->authorize('store', Message::class);
+        $message = Message::create(
+            $request->only(['title', 'content', 'link'])
+        );
+        $message->is_active = 0;
+        $message->save();
```

Comment:
Le problème ici c'est que tu fais 2 appels en bdd, un avec `Message::create(...)`et un avec `$message->save()`. Le mieux c'est de tout faire en 1 appel, donc lors de la création du message : 
```suggestion
        $message = Message::create(array_merge(
            $request->only(['title', 'content', 'link']),
            ['is_active' => false]
        ));
```